content/sensu-enterprise/3.1/guides/using-ec2.md

```diff
@@ -0,0 +1,286 @@
+
+You'll need to download the following example policy documents to follow along:
+
+* [Sensu IAM policy][8]
+* [Sensu IAM role][9]
+
+1. Create the IAM policy
+{{< highlight shell >}}
+aws iam create-policy --policy-name sensu-ec2-iam-policy --policy-document file://sensu-ec2-iam-policy.json{{< /highlight >}}
+Note the ARN for this policy, needed for step 3
+
+2. Create the IAM role
+{{< highlight shell >}}
+aws iam create-role --role-name sensu-enterprise-integration --assume-role-policy-document file://sensu-ec2-iam-role.json{{< /highlight >}}
+
+3. Attach policy to role
+{{< highlight shell >}}
+aws iam attach-role-policy --role-name sensu-enterprise-integration --policy-arn "arn:aws:iam::11111111111:policy/sensu-ec2-iam-policy"{{< /highlight >}}
+
+Now that we've created a policy and a role for the integration, we'll need to attach it to our Sensu Enterprise instance. You can do this during step 3 of creating any instance via the EC2 console:
+
+![ec2_role_create][10]
+
+Now that we have the policy and role created, and the role attached to an instance, let's see this in action on a test instance:
+
+![ec2_test_gif][11]
+
+## Cross-Account IAM Roles
+
+### Definitions
+
+Trusting account -- an AWS account where Sensu Clients are running. An
+IAM 'read-only' role is created here allowing ec2:DescribeInstances
+action, with a policy that allows the trusted account's instance role
+to assume the read-only role.
+
+For the purposes of this document:
+
+* The AWS account number for the trusting account is assumed to be `111111111111`
+* The AWS CLI is assumed to be configured with a profile `trusting`
+
+Trusted account -- an AWS account where Sensu Enterprise is running. An
+EC2 instance is created here with an instance role attached allowing
+the instance to assume a role in the trusting account.
+
+For the purposes of this document:
+
+* The AWS account number for the trusted account is assumed to be `222222222222`
+* The AWS CLI is assumed to be configured with a profile `trusted`
+
+### Configuration
+
+You can download the following files to follow along:
+
+* [Cross Account RO Policy][12]
+* [Cross Account RO Role][13]
+* [Cross Account Assume Role Policy][14]
+* [Cross Account Assume Role Policy Role][15]
+
+#### Integration Configuration
+
+You'll need to configure the EC2 integration file at `/etc/sensu/conf.d/ec2.json` so that it knows which account should be used when querying the API. See the example below:
+
+{{< highlight json >}}{
+  "ec2": {
+    "accounts": [
+      {
+        "name": "sensu-testing",
+        "role_arn": "arn:aws:iam::111111111111:role/cross-account-readonly"
+      }
+    ]
+  }
+}{{< /highlight >}}
+
+Where the account in the ARN above is the account that will be queried. 
+
+#### Client Configuration
+
+Clients will also need to be configured to use a specific account. This is done by the `"name"` specified in `/etc/sensu/conf.d/ec2.json`. See the example below:
```

Comment:
> This is done by the `"name"` specified in...

I think it should be the client.ec2.account attribute

How about this?

"This is done by the "account" attribute specified in `client.ec2` scope of `/etc/sensu/conf.d/ec2.json`"